docs/manuals/user/index.md

```diff
@@ -239,6 +208,46 @@ found in any of the attribute metadata sections in the notification.
 reception) if none of the above options is present or none of the values found
 can actually be converted to a `datetime`.
 
+### Restrictions and Limitations
+
+- You cannot have two entity types with the same name but different
+capitalisation. E.g: `Preprocessor` and `preProcessor`. The same applies for
+attribute names of a given entity. I.e, attributes `hotSpot` and `hotspot`
+will be treated as the same. These are rare corner-cases, but it is worth
+keeping in mind this. Ultimately, the correct naming of types and attributes
+should respect the naming guidelines explained
+[here](http://fiware-datamodels.readthedocs.io/en/latest/guidelines/index.html).
+
+- Attributes metadata are still not being persisted. See [Issue 12](https://github.com/smartsdk/ngsi-timeseries-api/issues/12)
+
+- While support for multiple data in a single notification as been recently introduced
+  (See [PR 191](https://github.com/smartsdk/ngsi-timeseries-api/pull/191)),
+  The following limitations still apply: a error in a single data entity will invalidate
+  the all set. There is not optimisation for large message size.
+
+- Data are assumed to be consistent. If in your first notification
+  you used a given type of an attribute, following ones will be rejected if
+  use a different attribute.
```

Comment:
"if a different attribute type is used" maybe?

Reply:
i tried to improve the text